examples/language/llama2/scripts/benchmark_7B/hosts.txt

```diff
@@ -0,0 +1,4 @@
+192.168.0.38
+192.168.0.39
+192.168.0.40
+192.168.0.41
```

Comment:
Remove this file?

Reply:
BUT we still need a sample. Should I add this to README?

Reply:
Yes, should provide an example in README.

Reply:
The NCCL variables can go to README as well.